appium-dotnet-driver/Appium/Service/AppiumServiceBuilder.cs

```diff
@@ -475,7 +475,7 @@ private string Args
                 if (PathToLogFile != null)
                 {
                     argList.Add("--log");
-                    argList.Add(PathToLogFile);
+                    argList.Add($"\"{PathToLogFile}\"");
```

Comment:
this will simply fix the case for this particular argument. I'd say it is necessary to properly escape all arguments in the code block below: https://blogs.msdn.microsoft.com/twistylittlepassagesallalike/2011/04/23/everyone-quotes-command-line-arguments-the-wrong-way/